index.html

```diff
@@ -42,6 +40,185 @@ <h2 class="intro secondary-color">I am a full stack developer.</h2>
+            </div>
+        </div>
+        <div class="center font">
+            <div class="card">
+                <div class="cover"></div>
+                <div class="content">
+                    <h1 class="card-name">Multi-Post Stories Gain+Glory</h1>
+                    <div class="language-container">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">CSS</div>
+                        <div class="language">Javascript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="see-project font">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="center font">
+            <div class="card">
+                <div class="cover"></div>
+                <div class="content">
+                    <h1 class="card-name">Multi-Post Stories Gain+Glory</h1>
+                    <div class="language-container">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">CSS</div>
+                        <div class="language">Javascript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="see-project font">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="center font">
+            <div class="card">
+                <div class="cover"></div>
+                <div class="content">
+                    <h1 class="card-name">Multi-Post Stories Gain+Glory</h1>
+                    <div class="language-container">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">CSS</div>
+                        <div class="language">Javascript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="see-project font">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="center font">
+            <div class="card">
+                <div class="cover"></div>
+                <div class="content">
+                    <h1 class="card-name">Multi-Post Stories Gain+Glory</h1>
+                    <div class="language-container">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">CSS</div>
+                        <div class="language">Javascript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="see-project font">
+                        See Project
+                    </button>
+                </div>
+            </div>
+        </div>
```

Comment:
Kindly wrap this section in a `<section>` tag. This is for semantic reasons and to make your site SEO friendly. Take a look at this [Semantic](https://www.pluralsight.com/guides/semantic-html) HTML article

Reply:
Also, this section should use a `display:grid`. This is a project requirement.

Reply:
Kindly use an `<ul>` for the `<div class='language-container'>` instead